src/ints/int10.cpp

```diff
@@ -632,34 +797,45 @@ static Bitu INT10_Handler(void) {
 			break;
 		}
 		break;
+
 	case 0xf0:
 		INT10_EGA_RIL_ReadRegister(reg_bl, reg_dx);
 		break;
+
 	case 0xf1:
 		INT10_EGA_RIL_WriteRegister(reg_bl, reg_bh, reg_dx);
 		break;
+
 	case 0xf2:
 		INT10_EGA_RIL_ReadRegisterRange(reg_ch, reg_cl, reg_dx, SegPhys(es)+reg_bx);
 		break;
+
 	case 0xf3:
 		INT10_EGA_RIL_WriteRegisterRange(reg_ch, reg_cl, reg_dx, SegPhys(es)+reg_bx);
 		break;
+
 	case 0xf4:
 		INT10_EGA_RIL_ReadRegisterSet(reg_cx, SegPhys(es)+reg_bx);
 		break;
+
 	case 0xf5:
 		INT10_EGA_RIL_WriteRegisterSet(reg_cx, SegPhys(es)+reg_bx);
 		break;
+
 	case 0xfa: {
 		RealPt pt=INT10_EGA_RIL_GetVersionPt();
 		SegSet16(es,RealSegment(pt));
 		reg_bx=RealOffset(pt);
 		}
 		break;
+
 	case 0xff:
-		if (!warned_ff) LOG(LOG_INT10,LOG_NORMAL)("INT10:FF:Weird NC call");
-		warned_ff=true;
+		if (!warned_ff) {
+			LOG(LOG_INT10, LOG_NORMAL)("INT10: FF:Weird NC call");
+		}
+		warned_ff = true;
```

Comment:
```suggestion
		if (!warned_ff) {
			LOG(LOG_INT10, LOG_NORMAL)("INT10: FF:Weird NC call");
			warned_ff = true;
		}
```

Reply:
Okay, but bear in mind I'm just doing minimal cleanups/reformats as parts of this change. I intentionally did _not_ want to even begin to understand that code too much; this was just cosmetic cleanup.